test/ably/restsetup.py

```diff
@@ -6,6 +6,7 @@
 from ably.types.capability import Capability
 from ably.types.options import Options
 from ably.util.exceptions import AblyException
+from ably.realtime.realtime import AblyRealtime
```

Comment:
Given the context (file named `restsetup.py`) then I am left wondering whether this file needs renaming to cover both REST and Realtime, or whether these pieces of setup should not actually be living in a different file (`realtimesetup.py`?) - which better reflects the natural hierarchy whereby Realtime inherits from REST.

Reply:
This is a good point, we were aware of the mismatch but tbh I'm planning to migrate the realtime tests to the setup used in #310 in the near future so I would rather we didn't spend the time making this stuff pristine.

Reply:
ok, as you were then. 😉